src/services/filters.ts

```diff
@@ -46,6 +46,7 @@ export function getLabelOptions(labels: string[]) {
   return [{ label: 'All', value: ALL_VARIABLE_VALUE }, ...labelOptions];
 }
 export const FIELDS_TO_REMOVE = ['level_extracted', LEVEL_VARIABLE_VALUE, 'level'];
+export const LABELS_TO_REMOVE = [LEVEL_VARIABLE_VALUE, 'level'];
```

Comment:
I'm not super convinced we should remove `level`. If people have that as an indexed field, we should leave it as it's better performance than `detected_level`, imo.

Reply:
Also, I think we are missing `service_name` here, right? As we can currently exclude that, which throws a bunch of errors.

https://github.com/user-attachments/assets/5e1568ea-9950-408b-b7ac-4d9a3088a107

Reply:
it's a bit odd seeing both detected_level and level though